tests/jvm/src/test/scala/scala/meta/tests/semanticdb/ExpectSuite.scala

```diff
@@ -19,6 +19,7 @@ import org.scalatest.FunSuite
 import org.scalatest.FunSuiteLike
 
 class ExpectSuite extends FunSuite with DiffAssertions {
+  override def trimLines: Boolean = false
```

Comment:
AppVeyor isn't gonna like this.

Reply:
If it's unhappy then I will fix that instead of continue living with trimmed diffs, it's so much easier on the eye when lines are not trimmed. I suspect `replace("\r\n", "\n")` will do the trick, I had to do something similar in scalafix to please AppVeyor.

Reply:
Sounds good.